packages/env/README.md

```diff
@@ -11,6 +11,7 @@
 ## What you get with this project
 
 - `@theforeman/env/babel` - Adds theforeman babel dev configuration to your project.
+- `@theforeman/env/test` - Adds theforeman tests configuration to your project.
 - WIP: `@theforeman/env/lint` - Adds theforeman linting tools to you project.
 - WIP: `@theforeman/env/test` - Adds theforeman testing tools to you project.
```

Comment:
Remove this line please. Also, IMO this description better (`Adds theforeman testing tools to you project.`)

Reply:
I removed also ` WIP: @theforeman/env/lint`